docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -618,6 +618,27 @@ The master supports the following configuration settings:
             such as "30s", "1h", or "1m30s". Valid time units are "s", "m", "h". The default value
             is ``5m``.
 
+      -  ``type: hpc``: Specifies running dynamic agents on HPC clusters. (*Required*)
```

Comment:
Specifies a custom resource pool that submits work to an underlying Slurm/PBS partition on an HPC cluster. (*Required*)

One resource pool is automatically created for each partition/queue on an HPC cluster.   This provider enables creation of additional resource pools with different submission options to those partitions/queues.

Reply:
Done